### ISSUE TITLE
Weird MMC1 compatibility issue with the base Everdrive N8

### ISSUE THREAD

**Issue by LeKukie** [pellsson/smb#47]
This issue does not seem to happen in any emulator, and it doesn't happen on the Everdrive N8 PRO nor the Powerpak. The issue started happening when the change of graphics bankswitching was made to account for more faces in the title screen, instead of the old way of rewriting them. As you'll see, it affects the title screen graphics, the whole entirety of the 2j graphics, and how peach's graphics are loaded (only the palette is loaded but mario's graphics are the same). SMB1's graphics are unaffected by this issue, other than selecting peach in the title screen since that is caused by the bankswitching issue.

I have tested the 2 latest OS versions of the base EDN8, aswell as v1.16 which used to be the most used version in the past. All of them had the exact same behaviour. Validusername16 has also tested the 6.0 build in his base EDN8 and the behaviour is also the same.

One somewhat curious thing is that, a beta of version 2.2 that was reported to use the bankswitching feature is not broken at all. Not sure if that has to do with the code using different features (or revisions) of the MMC1 related to bankswitching, or if the amount of banks is screwing up with things.

[https://youtu.be/HKoj6LJcs7A](url) This video shows everything I talked about here.

**Comment by pellsson** [comment]
Yeah expect the master-branch (and all other branches for that matter) to be pretty borked atm :) Everything is highly work in progress.
